sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaFormat.cs

```diff
@@ -20,9 +20,15 @@ public SchemaFormat(string value)
         }
 
         private const string AvroValue = "Avro";
+        private const string JsonValue = "JSON";
```

Comment:
Not sure about the capitalizations here, should it be JSONValue="JSON" or JsonValue="Json" instead?

Reply:
I think `JSON` make sense here; that's the [official capitalization](https://www.json.org/json-en.html).